README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
```

Comment:
Yeah, I see. In this case, it's a tradeoff between convenience and adding more stuff to read. Besides, the README is pretty big already and there are some topics that direct you to links to reduce cluttering.

There are also arguments [against copy/pasting code](https://medium.freecodecamp.com/the-benefits-of-typing-instead-of-copying-54ed734ad849), which might also be useful to people more than the convenience copy/paste gives.

Do you think it's still worth adding this information here?